docs/guides/creatingadls.md

```diff
@@ -0,0 +1,140 @@
+# Creating a Bridge With a Dead Letter Sink (DLS)
+
+## What is a Dead Letter Sink?
+
+Knative Eventing provides various configuration parameters to control the delivery of events in case of failure. For instance, you can decide to retry sending events that failed to be consumed, and if this didn't work you can decide to forward those events to a dead letter sink.
+
+## Implementing a Dead Letter Sink
+
+There are two ways to implement a dead letter sink, either by use of a Subscription or by use of a Broker.
```

Comment:
If I understand correctly, you can do DLS both on Channels (using Subscriptions) and Brokers (directly defined on the Broker). If thats the case, could it worth mentioning Channels here to make sure users not fully familiar with the Channel / Broker split can follow better?